_posts/2017-05-10-fastlane-travis-weekly-deploys.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post_longform
+title: Deploying your app on a weekly basis via fastlane + Travis CI
+date: 2017-05-10
+author: orta
+categories: [mobile, ios, emission, fastlane]
+---
+
+We have a few apps now, but one of them isn't really used by anyone other than developers. This is [our React Native host app][em_app]. We built our React Native components [as a library][how_emission] to be consumed by our main app, and in the process we have a separate app that just acts as a host for the React Native components. This app is often updated, but never deployed to beta users - where it could be of use.
```

Comment:
> and in the process we have a separate app

Tense sounds strange. Maybe "as a result of that process, we have a separate app"